src/workloads/scale/BulkLoading.yml

```diff
@@ -0,0 +1,287 @@
+  # We evaluated the impact of different batch sizes and the document throughput
+  # does not appear to be impacted by the batch size.
+  - &WriteOp { WriteCommand: insertOne, Document: *Document }
+  - &WriteOps10x [
+      *WriteOp, *WriteOp, *WriteOp, *WriteOp, *WriteOp,
+      *WriteOp, *WriteOp, *WriteOp, *WriteOp, *WriteOp,
+    ]
+  - &WriteOps100x {
+      ^FlattenOnce:
+        [
+          *WriteOps10x, *WriteOps10x, *WriteOps10x, *WriteOps10x, *WriteOps10x,
+          *WriteOps10x, *WriteOps10x, *WriteOps10x, *WriteOps10x, *WriteOps10x,
+        ],
+    }
+  - &UpsertOp {
+      WriteCommand: updateOne,
+      Filter: { sensorId: { ^RandomInt: { min: 0, max: 200000000 } } },
+      Update:
+        {
+          $set:
+            { eventId: 0, dataField1: { ^FastRandomString: { length: 10 } } },
+        },
+      OperationOptions: { Upsert: true },
+    }
+  - &UpsertOps10x [
+      *UpsertOp, *UpsertOp, *UpsertOp, *UpsertOp, *UpsertOp,
+      *UpsertOp, *UpsertOp, *UpsertOp, *UpsertOp, *UpsertOp,
+    ]
+  - &UpsertOps100x {
+      ^FlattenOnce:
+        [
+          *UpsertOps10x, *UpsertOps10x, *UpsertOps10x, *UpsertOps10x, *UpsertOps10x,
+          *UpsertOps10x, *UpsertOps10x, *UpsertOps10x, *UpsertOps10x, *UpsertOps10x,
+        ],
+    }
+
+ActorTemplates:
+  - TemplateName: BulkWriterTemplate
+    Config:
+      Name: { ^Parameter: { Name: "Name", Default: "unset" } }
+      Type: CrudActor
+      Threads: { ^Parameter: { Name: "Threads", Default: "unset" } }
+      Phases:
+        OnlyActiveInPhases:
+          Active:
+            { ^Parameter: { Name: "OnlyActiveInPhase", Default: "unset" } }
+          NopInPhasesUpTo: *NumPhases
+          PhaseConfig:
+            Duration: { ^Parameter: { Name: "Duration", Default: "unset" } }
+            Database: *DbName
+            Collection: *CollectionName
+            Operations:
+              - OperationName: bulkWrite
+                OperationCommand:
+                  WriteOperations:
+                    { ^Parameter: { Name: "Operations", Default: "unset" } }
+  - TemplateName: NormalWriterTemplate
+    Config:
+      Name: { ^Parameter: { Name: "Name", Default: "unset" } }
+      Type: CrudActor
+      Threads: { ^Parameter: { Name: "Threads", Default: "unset" } }
+      Phases:
+        OnlyActiveInPhases:
+          Active:
+            { ^Parameter: { Name: "OnlyActiveInPhase", Default: "unset" } }
+          NopInPhasesUpTo: *NumPhases
+          PhaseConfig:
+            Duration: { ^Parameter: { Name: "Duration", Default: "unset" } }
+            Database: *DbName
+            Collection: *CollectionName
+            Operations:
+              - OperationName: insertOne
+                OperationCommand:
+                  Document: *Document
+
+Actors:
+  # This loads about 35 GB of data to warm up the DB
+  - ActorFromTemplate:
+      TemplateName: BulkWriterTemplate
+      TemplateParameters:
+        Name: Warmup
```

Comment:
This isn't really warmup, it's Loading based on your phases descriptions earlier.

Question: Should we have a warmup phase?

Reply:
The intention here was to include the term Warmup, to make sure this metric gets ignored by the build barons. I've renamed the parameter to WarmupLoadData to better describe what the actor is used for.

Reply:
oh github hid this response as the block was outdated .... but I think we should fix the way data is ignored rather than using a work around like this.